tokio/src/process/mod.rs

```diff
@@ -1164,7 +1164,11 @@ impl Child {
 
     /// Forces the child to exit.
     ///
-    /// This is equivalent to sending a `SIGKILL` on unix platforms.
+    /// This is equivalent to sending a `SIGKILL` on unix platforms followed by `wait`.
+    ///
+    /// Note: behavior of this function is different from std version of
+    /// [`Child::kill`](std::process::Child::kill): std version does `wait`.
+    /// Tokio equivalent of std `Child::kill` is [`start_kill`](Child::start_kill).
```

Comment:
nit: I think the phrasing could be a bit simplified. Like:
```
Note: The equivalent of `Child::kill` in the standard library is [`start_kill`](Child::start_kill).
```

Reply:
I cannot comprehend your version (what side is what); rephrased.

Reply:
How about this:
```
Note: For an equivalent of `Child::kill` in the standard library, use [`start_kill`](Child::start_kill).
```

Reply:
Like. Another iteration.